WooCommerce/Classes/ViewRelated/Products/ProductsViewController.swift

```diff
@@ -500,11 +508,13 @@ private extension ProductsViewController {
     }
 
     func configureResultsController(_ resultsController: ResultsController<StorageProduct>, onReload: @escaping () -> Void) {
-        resultsController.onDidChangeContent = {
+        resultsController.onDidChangeContent = { [weak self] in
+            self?.showOrHideToolBar(resultController: resultsController)
             onReload()
         }
 
-        resultsController.onDidResetContent = {
+        resultsController.onDidResetContent = { [weak self] in
+            self?.showOrHideToolBar(resultController: resultsController)
             onReload()
```

Comment:
I think you could move `showOrHideToolBar` to the `onReload` declaration and avoid a little bit of repetition, right?

Reply:
Totally agree, moved to the declaration! thanks!